root/documentation/vep.conf

```diff
@@ -1280,7 +1530,12 @@
         type=Boolean
         description=Include HGVS nomenclature based on Ensembl stable identifiers
         default=0
-      </hgvs> 
+      </hgvs>
+      <ambiguous_hgvs>
+        type=Boolean
```

Comment:
Is the output sufficiently clear when multiple ambiguous variants are submitted? Is there an upper limit on the number of variants which can be converted in a single request

Reply:
> Is there an upper limit on the number of variants which can be converted in a single request

There is no upper limit.

> Is the output sufficiently clear when multiple ambiguous variants are submitted?

Running a POST request for `ENSP00000492425.1:p.Met1Ala` (non-ambiguous) and `ENSP00000492425.1:p.Met1Leu` (ambiguous):

`curl -d '{ "hgvs_notations" : ["ENSP00000492425.1:p.Met1Ala", "ENSP00000492425.1:p.Met1Leu" ], "ambiguous_hgvs" : 1 }' -H "Content-Type: application/json" -X POST http://wp-np2-11.ebi.ac.uk:9300/vep/human/hgvs`

The following output is returned (consequences were truncated for simplicity). I think the output is clear, it will return an entry for each ambiguous HGVS (`ENSP00000492425.1:p.Met1Leu ` in this example) with different alternative alleles:

```json
[
  {
    "seq_region_name": "1",
    "start": 148748953,
    "end": 148748954,
    "input": "ENSP00000492425.1:p.Met1Ala",
    "most_severe_consequence": "start_lost",
    "strand": 1,
    "id": "ENSP00000492425.1:p.Met1Ala",
    "assembly_name": "GRCh38",
    "motif_feature_consequences": [ ... ],
    "allele_string": "AT/GC",
    "regulatory_feature_consequences": [ ... ],
    "transcript_consequences": [ ... ]
  },
  {
    "input": "ENSP00000492425.1:p.Met1Leu",
    "most_severe_consequence": "start_lost",
    "strand": 1,
    "assembly_name": "GRCh38",
    "id": "ENSP00000492425.1:p.Met1Leu",
    "start": 148748953,
    "seq_region_name": "1",
    "end": 148748953,
    "regulatory_feature_consequences": [ ... ],
    "transcript_consequences": [ ... ],
    "motif_feature_consequences": [ ... ],
    "allele_string": "A/C"
  },
  {
    "transcript_consequences": [ ... ],
    "regulatory_feature_consequences": [ ... ],
    "allele_string": "A/T",
    "motif_feature_consequences": [ ... ],
    "id": "ENSP00000492425.1:p.Met1Leu",
    "assembly_name": "GRCh38",
    "input": "ENSP00000492425.1:p.Met1Leu",
    "most_severe_consequence": "start_lost",
    "strand": 1,
    "end": 148748953,
    "start": 148748953,
    "seq_region_name": "1"
  }
]
```